suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -350,7 +350,10 @@ export const selectDevicelessAccounts = (state: AccountsRootState & DeviceRootSt
     ) as Account[];
 };
 
-export const selectAreAllDevicesDisconnectedOrAccountless = (
+/**
+ * Return true only if there is just a portfolio tracker device and that has empty accounts.
+ */
+export const selectIsDeviceDiscoveryEmptyAndNoPhysicalDeviceConnected = (
```

Comment:
If this really does what its comment says, what about renaming it to `selectIsEmptyPortfolioTracker` to make it even more readable?